src/charm.py

```diff
@@ -5,6 +5,7 @@
 import json
 import logging
 import subprocess
+import time
 from subprocess import check_call
```

Comment:
Could we use one style with subprocess: or `subprocess.check_call` or `from subprocess import check_call, check_output`

Reply:
sure, I will change it to `from subprocess import check_call, check_output`